Sage/carbon
# refactor(app-wrapper): add deprecation warning (FE-3649)

### Proposed behaviour

- Remove any remaining references of `AppWrapper`  in other components and storybook docs, and replace with `Box`.
- Add deprecation warning to `AppWrapper`'s code and story file

### Current behaviour

`AppWrapper` has no deprecation warning in its code despite plans to remove it. It's also used in a couple of stories.

### Checklist




#### QA


### Testing instructions

Check for any major regressions in layout in stories which will now use `Box` instead of `AppWrapper`.
